index.js

```diff
@@ -23,10 +23,23 @@ function handleRelativePath(_relativePath) {
 }
 
 module.exports = walkSync;
-function walkSync (baseDir, _options, _relativePath) {
+function walkSync(baseDir, _options) {
```

Comment:
I like that we're extracting the `_walkSync` function and dropping the internal third parameter here. :+1: